plugins/woocommerce/includes/admin/meta-boxes/views/html-product-attribute.php

```diff
@@ -21,7 +21,7 @@
 							<strong><?php echo wc_attribute_label( $attribute->get_name() ); ?></strong>
 							<input type="hidden" name="attribute_names[<?php echo esc_attr( $i ); ?>]" value="<?php echo esc_attr( $attribute->get_name() ); ?>" />
 						<?php else : ?>
-							<input type="text" class="attribute_name" name="attribute_names[<?php echo esc_attr( $i ); ?>]" value="<?php echo esc_attr( $attribute->get_name() ); ?>" />
+							<input type="text" class="attribute_name" name="attribute_names[<?php echo esc_attr( $i ); ?>]" value="<?php echo esc_attr( $attribute->get_name() ); ?>" placeholder="<?php esc_attr_e( 'New attribute', 'woocommerce' ); ?>" />
```

Comment:
As far as I understood from [this comment](https://github.com/woocommerce/woocommerce/pull/35904#issuecomment-1344342733), it should say `e.g. Fabric or Brand` instead of `New attribute`.